kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/wlst/createWLSDomain.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
```

Comment:
We should be using off-line WLST - not on-line WLST to create the domain. No need to start an admin server.